js/apps/system/_admin/aardvark/APP/react/yarn.lock

```diff
@@ -4038,9 +4062,9 @@ anymatch@~3.1.2:
     picomatch "^2.0.4"
 
 arangojs@^7.5.0:
-  version "7.5.0"
-  resolved "https://registry.npmjs.org/arangojs/-/arangojs-7.5.0.tgz"
-  integrity sha512-9taMv73bo0O/bpFeqcT4xnYzu5yg+UZ9qGQ9SCAQXnKZZcOAwP1TixvOBv21il7XIzX28RCVoH+wp/u8Ajb8+Q==
+  version "7.8.0"
```

Comment:
this is another version change which seems okay. Strange that your yarn didn't do this @pluma